### PR TITLE
Highlight active context / menu in ContextSelector

### PR DESCRIPTION
**Pull request by josemigallas**
**What this PR does / why we need it**:

According to the mockup, the active menu AKA selected context has to be highlighted.
Also, when having only 1 service the ContextSelector is not hidden anymore. Only the `input` element is hidden instead.

ApiSearch and ContextSelector have fused

**Verification steps** 

1. Go to Dashboard, Dashboard only should be highlighted.
2. Go to Audience, Audience only should be highlighted regardless of the submenu selected.
3. Go to any Service, that service only should be highlighed regardless of the submenu selected.
4. When having 1 service only, the dropdown should be available but no input element inside.

**Special notes for your reviewer**:

The _Overview_ page of any service with make `ContextSelector` fail for now.

### CODE REVIEW

File: app/javascript/src/Navigation/components/ContextSelector.jsx [hunk 1]
```diff
@@ -5,28 +5,95 @@ import 'core-js/modules/es6.map'
...
+    if (apis.length < 2) {
+      return null
+    }
+
+    return (
+      <li className="PopNavigation-listItem nav-search-widget docs-search" id="context-widget">
+        <input
+          onChange={e => this.onFilterChange(e)}
+          type="search"
+          className="docs-search-input"
```
Comment (mayorova) [inline app/javascript/src/Navigation/components/ContextSelector.jsx]: Doesn't belong to this PR really, but just wondering if it would make sense to rename this class to something more generic, as it's not only used for Docs search now. Maybe `search-input`? :) cc @thomasmaas

Comment (thomasmaas) [reply app/javascript/src/Navigation/components/ContextSelector.jsx]: yes but no time. next iteration.
